### ISSUE TITLE
Thread navigation 

### ISSUE THREAD

**Issue by davidoskky** [thundernest/k-9#6127]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.000

### Problem you are trying to solve

When I open one email I replied to, there is no straightforward way to open my reply and check what I wrote.
A similar problem appears when you're trying to get to the original mail or to your previous reply.

### Suggested solution

A way to navigate a thread would be very useful.
Once you open an email, there should be a way to list the mails in the thread and open one of those.
Back/forward buttons might work, however this wouldn't work when an email received more than one reply.
In thunderbird for example a wonderful functionality is offered by the extension [threadvis](https://github.com/threadvis/ThreadVis).
Surely such a complex feature cannot be implemented in a phone application, but maybe something like an indented drop-down menu might work.

**Comment by ByteHamster** [comment]
Duplicate of #5601

**Comment by davidoskky** [comment]
Not really, I want to be able to easily find mails within one thread, not mute and hide the thread.
